bibliopixel/drivers/serial_driver.py

```diff
@@ -80,7 +80,11 @@ class DriverSerial(DriverBase):
     deviceIDS = {}
     deviceVers = []
 
-    def __init__(self, type, num, dev="", c_order=ChannelOrder.RGB, SPISpeed=2, gamma=None, restart_timeout=3, deviceID=None, hardwareID="1D50:60AB"):
+    def __init__(self, type, num, dev="",
+                 c_order=ChannelOrder.RGB, SPISpeed=2,
+                 gamma=None, restart_timeout=3,
+                 deviceID=None, hardwareID="1D50:60AB",
+                 baudrate=921600):
```

Comment:
Clearly not so standard that I knew about it!

This is very un-[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself) but we can do this later.  Merging!